lib/ret/hub.ex

```diff
@@ -33,6 +33,7 @@ defmodule Ret.Hub do
     field(:creator_assignment_token, :string)
     field(:embed_token, :string)
     field(:embedded, :boolean)
+    field(:permissions, :integer)
```

Comment:
worth considering a rename to `member_permissions` or `default_permissions` -- this could get a bit tricky to understand if we do end up having more complex policies down the road